coding-guidelines.md

```diff
@@ -0,0 +1,24 @@
+# Coding Guidelines
```

Comment:
By coding guidelines, I meant https://google.github.io/styleguide/jsguide.html not the wikipedia page.

Reply:
I added it as a citation. I've also added the above link now.